sql/src/main/java/org/apache/druid/sql/calcite/schema/MetadataSegmentView.java

```diff
@@ -195,7 +206,7 @@ private void poll()
         sb.append("datasources=").append(ds).append("&");
       }
       sb.setLength(sb.length() - 1);
-      query = "/druid/coordinator/v1/metadata/segments?" + sb;
+      query = "/druid/coordinator/v1/metadata/segments?includeOvershadowedStatus&" + sb;
```

Comment:
yes, i think so, this will be only used if there are non empty `watchedDataSources` set (let's say it contains a datasource name "dummy") then the URL would look like `/druid/coordinator/v1/metadata/segments?includeOvershadowedStatus&datasources=dummy` etc.